### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by frankiefab100**
Add `@frankiefab100` Github README file.

### CODE REVIEW

File: Readmes/Franklin Ohaegbulam/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<h1 align="center">Salut 👋, I'm Franklin</h1>
+<h3 align="center">A Multidisciplinary Designer and Software Developer</h3>
+
+<p align="left"> <img src="https://komarev.com/ghpvc/?username=frankiefab100&label=Profile%20views&color=0e75b6&style=flat" alt="frankiefab100" /> </p>
```
Comment (Panquesito7) [inline Readmes/Franklin Ohaegbulam/README.md:4]: I recommend removing the profile views badge as it's inaccurate.
```suggestion
```

File: Readmes/Franklin Ohaegbulam/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+- 📒 I’m currently learning **ReactJS, NodeJS and Solidity** and working on side projects.
+
+- 🤝🏻 I’m passionate about bridging the gap between design and development and look forward to collaborating to **Open source Projects**.
+
+- 🌎 I'm available for any Design or UI Engineer roles.
+
+- 💬 Feel free to reach out: DM [@frankiefab100](https://twitter.com/frankiefab100)
```
Comment (Panquesito7) [inline Readmes/Franklin Ohaegbulam/README.md:12]:
```suggestion
- 💬 Feel free to reach me out at my Twitter: [@frankiefab100](https://twitter.com/frankiefab100)
```

File: Readmes/Franklin Ohaegbulam/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+- [Ten (10) Reasons you should contribute to free and open source projects](https://frankiefab.hashnode.dev/ten-10-reasons-you-should-contribute-to-free-and-open-source-projects-ckhj55rzq00ud7ys193oid5ia)
+- [Web Optimization | An Overview of Search Engine Optimization](https://dev.to/frankiefab100/web-optimization-an-overview-of-search-engine-optimization-seo-4olk)
+
+➡️ [More articles...](https://frankiefab.hashnode.dev)
+<!-- BLOG-POST-LIST:END -->
+
+---
+
+<p align="left">
+<h3 align="left"> Let's Connect:</h3>
```
Comment (Panquesito7) [inline Readmes/Franklin Ohaegbulam/README.md:30]:
```suggestion
<h3 align="left"> Let's Connect</h3>
```

File: Readmes/Franklin Ohaegbulam/README.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+<a href="https://instagram.com/frankiefab100" target="blank"><img align="center" src="https://cdn.jsdelivr.net/npm/simple-icons@3.0.1/icons/instagram.svg" alt="@frankiefab100" height="30" width="40" /></a>
+<a href="https://dribbble.com/frankiefab100" target="blank"><img align="center" src="https://cdn.jsdelivr.net/npm/simple-icons@3.0.1/icons/dribbble.svg" alt="frankiefab100" height="30" width="40" /></a>
+<a href="https://www.behance.net/frankiefab100" target="blank"><img align="center" src="https://cdn.jsdelivr.net/npm/simple-icons@3.0.1/icons/behance.svg" alt="@frankiefab100" height="30" width="40" /></a>
+<a href="https://medium.com/@frankiefab100" target="blank"><img align="center" src="https://cdn.jsdelivr.net/npm/simple-icons@3.0.1/icons/medium.svg" alt="@frankiefab100" height="30" width="40" /></a>
+<a href="https://frankiefab100.disha.page" target="_blank"><img align="center" src="https://raw.githubusercontent.com/iconic/open-iconic/master/svg/globe.svg" alt="frankiefab100 personal website" height="30" width="40" /></a>
+</p>
+
+<h3 align="left">Languages and Tools:</h3>
```
Comment (Panquesito7) [inline Readmes/Franklin Ohaegbulam/README.md:43]:
```suggestion
<h3 align="left">Languages and Tools</h3>
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by frankiefab100** [comment]
Alright

**Review by Panquesito7 — COMMENTED** [review]
There are still some of my suggestions above not being applied. If you don't want them, however, please let me know. 🙂

**Comment by frankiefab100** [comment]
@Panquesito7 I hope I got it right this time?🙂

**Comment by Greeshma2903** [comment]
This looks great @frankiefab100, going ahead with the merge 🚀

**Review by Panquesito7 — COMMENTED** [review]
LGTM! 🚀

**Pull request merged**